### DISCUSSION TITLE
Is there a good way to not send pressed keycodes in insert Mode?

### DISCUSSION THREAD

**Discussion by lucordes** [pit-ray/win-vind#170] [General]
I use AHK to open my file explorer instead of the usual one. I with to only use win-vind in the future and not open the default explorer.
If I use something like this: 
imap <Win-e> <to_command>execute C:\Program Files\Double Commander\doublecmd.exe<cr>
it works fine but also opens the default file explorer. How can I not pass <Win-e> to the system?
Thank you very much. <3

**Comment by pit-ray** [comment]
Sorry for my late reply.

Are you saying that your definition of a map is the following?
```vim
imap <win-e> <to_command>execute C:\Program Files\Double Commander\doublecmd.exe
```

In the insert mode, the pressed key is passed to the system without absorbing.
Currently, if the input key is a single one, the messages can be stoppped, but the combination keys  are not allowed due to timing mismatches in pressing them.

Thanks.
